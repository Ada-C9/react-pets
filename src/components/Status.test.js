// src/components/NewPetForm.test.js
import React from 'react';
import Status from './Status';
import { shallow } from 'enzyme';

describe('Status', () => {
  test('matches snapshot with error type', () => {
    const statusComponent = shallow(
      <Status
        message="test status"
        type="error"
      />
    );

    expect(statusComponent).toMatchSnapshot();

    statusComponent.unmount();
  });

  test('matches snapshot with non-error type', () => {
    const statusComponent = shallow(
      <Status
        message="test status"
        type="success"
      />
    );

    expect(statusComponent).toMatchSnapshot();

    statusComponent.unmount();
  });
});
