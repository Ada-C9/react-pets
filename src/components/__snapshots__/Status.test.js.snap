// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status matches snapshot with error type 1`] = `
<section
  className="status error"
>
  There was a problem: 
  test status
</section>
`;

exports[`Status matches snapshot with non-error type 1`] = `
<section
  className="status success"
>
  test status
</section>
`;
